sdks/python/apache_beam/options/pipeline_options.py

```diff
@@ -554,6 +554,12 @@ def _add_argparse_args(cls, parser):
          'worker will install the resulting package before running any custom '
          'code.'))
     parser.add_argument(
+        '--beam_plugins',
+        default=None,
+        help=
+        ('Bootstrap the python process before executing any code by importing '
```

Comment:
This is also in kind of a weird place because you don't really expect users to specify this--you're just using the pipeline option to transport this module information.  (not sure if there's a good way to clarify)